docs/content/foundations/layout-positioning/layout-grid.mdx

```diff
@@ -0,0 +1,356 @@
+
+- `grid-container` can also accept any breakpoint width, like `tablet-lg` or `widescreen`. Set the default max width with `$theme-grid-container-max-width` in your [USWDS settings configuration](/foundations/utilities-overview) using [unit tokens](/foundations/spacing-units).
+
+- By default, `grid-container` has a `padding-x` of **`2 units`** at narrow widths, and a `padding-x` of **`4 units`** at `desktop` and wider. Control these values by customizing `$theme-site-margins-mobile-width`, `$theme-site-margins-width` and `$theme-site-margins-breakpoint` in your [USWDS settings configuration](/foundations/utilities-overview) using [unit tokens](/foundations/spacing-units).
+
+<hr className="measure-6 margin-left-0" />
+
+**Rows**: Columns must have a `grid-row` as a parent.
+
+<hr className="measure-6 margin-left-0" />
+
+**Columns**: `grid-col-[1-12]` indicates the number of columns the item spans out of a possible 12 per row. So, if you want three equal-width columns across, use `grid-col-4` for each item.
+
+### Additional Functionality
+
+<hr className="measure-6 margin-left-0" />
+
+**Equal-width columns**: With flexbox, grid columns without a specified width will display as equal-width columns. For example, four instances of `grid-col` will display as one-quarter-width columns across all sizes. Refer to the [auto-layout columns](#auto-layout-columns) section for more examples.
+
+<hr className="measure-6 margin-left-0" />
+
+**Gutters**: Rows and columns don’t have any gutters by default, but gutters can be added by including `grid-gap-sm`, `grid-gap`, or `grid-gap-lg` at the row level. Refer to [gutters](#gutters) for more info.
+
+<hr className="measure-6 margin-left-0" />
+
+**Media queries**: Grid breakpoints are based on minimum-width media queries, meaning they apply to that specific width and all greater widths (e.g., `tablet:col-4` applies to `tablet`, `desktop`, and `widescreen` devices but not at `mobile-lg` or any width below the `tablet` breakpoint). Refer to [responsive variants](#responsive-variants) for a full list.
+
+<hr className="measure-6 margin-left-0" />
+
+**Sass mixins**: You can use predefined grid classes (like grid-col-4) for presentational markup or Sass mixins for more semantic markup.
+
+<h2 id="responsive-variants">Responsive Variants</h2>
+
+| Width        | Size      | Utility Class         | Columns | Gutters | Nestable |
+| :----------- | :-------- | :-------------------- | :------ | :------ | :------- |
+| Smallest     | >= 0      | `.grid-col`           | 12      | 0       | Yes      |
+| Mobile Large | >= 480px  | `.mobile-lg:grid-col` | 12      | 0       | Yes      |
+| Tablet       | >= 640px  | `.tablet:grid-col`    | 12      | 0       | Yes      |
+| Desktop      | >= 1024px | `.desktop:grid-col`   | 12      | 0       | Yes      |
```

Comment:
Since we added tablet-lg we should add that to this list. (I am pretty sure we did not turn on Desktop)

Reply:
So remove Desktop and replace it appropriately with Tablet Large in this table ?

Reply:
I meant Widescreen.